#src/scss/lib/variables.scss

```
@@ -0,0 +1,7 @@
+$fontHeading: "Playfair Display", sans-serif, serif;
+$fontPrimary: "Mulish", sans-serif, serif;
+$fontSecondary: "Inter, sans-serif, serif";
+$colorLight: #7D7987;
+$colorDark: #202336;
+$colorWhite: #ffffff;
+$colorPrimary: #FB8F1D;
```

You may have lots of variables in bigger projects, so you better start split and organize them more accurately, for example:
```suggestion
/* FONT VARIABLES START */
$fontHeading: "Playfair Display", sans-serif, serif;
$fontPrimary: "Mulish", sans-serif, serif;
$fontSecondary: "Inter, sans-serif, serif";
/* FONT VARIABLES END */

/* COLOR VARIABLES START */
$colorLight: #7D7987;
$colorDark: #202336;
$colorWhite: #ffffff;
$colorPrimary: #FB8F1D;
/* COLOR VARIABLES END */
```
so you navigate through file easily when it becomes more than 1-2 screen long lines